src/Corax/Pipeline/LowerCaseTransformer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Corax.Pipeline
+{
+    public struct LowerCaseTransformer : ITransformer
+    {
+        public void Dispose() { }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public int Transform(ReadOnlySpan<byte> source, ReadOnlySpan<Token> tokens, ref Span<byte> dest, ref Span<Token> destTokens)
+        {
+            for (int i = 0; i < source.Length; i++)
```

Comment:
This is wrong, need to also handle Unicode.
Can do that in two stages, optimally, try ASCII mode, otherwise, using the encoder.

Reply:
I know this is related to what we talked the other day. This was just an example to understand how that would work.